tasks/buildsources.js

```diff
@@ -107,6 +107,15 @@ module.exports = function(grunt) {
 				config.rawSource = '\n// '+featureName + '\n' + config.rawSource;
 
 				var featurePath = path.join(destFolder, featureName+'.json');
+                var featureDir = path.dirname(featurePath);
```

Comment:
The indentation doesn't seem correct, would you be able o change it from spaces to tabs?